src/picosvg/svg_transform.py

```diff
@@ -65,6 +65,9 @@ def degenerate():
     def fromstring(raw_transform):
         return parse_svg_transform(raw_transform)
 
+    def tostring(self):
+        return f'matrix({" ".join(str(v) for v in self)})'
```

Comment:
hm. @anthrotype I wonder if this should be ntos [from svg_meta] not just str

Reply:
good point, done